modules/using-cluster-compare-live-cluster.adoc

```diff
@@ -0,0 +1,81 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="using-cluster-compare-live-cluster_{context}"]
+= Using the cluster-compare plugin with a live cluster
+
+You can use the `cluster-compare` plugin to compare a reference configuration with configuration custom resources (CRs) from a live cluster. 
+
+Validate live cluster configurations to ensure compliance with reference configurations during design, development, or testing scenarios.
+
+[NOTE]
+====
+Use the `cluster-compare` plugin with live clusters in non-production environments only. For production environments, use the plugin with `must-gather` data.
+====
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+* You downloaded the `cluster-compare` plugin and include it in your `PATH` environment variable.
+
+* You have access to a reference configuration. 
+
+.Procedure
+
+* Run the `cluster-compare` plugin by using the following command:
++
+[source,terminal]
+----
+$ oc cluster-compare -r <path/to/reference/config>/metadata.yaml <1>
```

Comment:
There is a `<1>` in the code block, but there isn't a reference to it in the explanation that immediately follows it. Suggest removing `<1>`